.github/workflows/test.yml

```diff
@@ -61,6 +61,12 @@ jobs:
       run: |
         pip install -U pytest-rerunfailures
       if: ${{ matrix.ckan-version == '2.9' }}
+    - name: Setup other extensions
+      run: |
+        git clone https://github.com/OpenGov-OpenData/ckanext-harvest
```

Comment:
would you mind replacing this with actions/checkout@v4 ? it is optimized for CI/CD, as, by default, it performs a shallow clone